sdk/openai/azure-ai-openai/tsp-location.yaml

```diff
@@ -1,5 +1,5 @@
 directory: specification/cognitiveservices/OpenAI.Inference
 additionalDirectories:
     - specification/cognitiveservices/OpenAI.Authoring
-commit: db23874c147db173933476b3b352cbf12abe84a9
```

Comment:
Even with the latest commit from azure-rest-api-specs but this PR won't add any non-naming changes from this PR https://github.com/Azure/azure-rest-api-specs/commit/82639d7ed6adbb64704f2f9db779eabee9d8fec3